utils/Constants.ts

```diff
@@ -6,6 +6,7 @@ export const MGR_ASSET_NAME = "MGR";
 export const ETH_ASSET_ID = new BN(1);
 export const ETH_ASSET_NAME = "ETH";
 export const mETH_ASSET_NAME = "mETH";
+export const DUMMY_POOL_ASSET_ID = new BN(2);
```

Comment:
I think I'm missing something, what is this?

Reply:
We use this token for microapps test https://github.com/mangata-finance/mangata-e2e/blob/ac130257cdf9523cbb57773c81e0ab590e82c522/test/microapps-prod/microapps-prod-main.pools.test.ts#L101